adback-solution-to-adblock/admin/partials/ad-back-admin-settings-display.php

```diff
@@ -52,6 +54,10 @@
                 <section style="background-color:transparent;">
                     <h4 class="header-section"><?php esc_html_e('Adback Account', 'adback-solution-to-adblock'); ?></h4>
                     <hr/>
+                    <div class="section-content">
+                        <p style="margin: auto !important;">Login: <a><?php echo $email ?></a></p>
+                        <i><?php _e('We sent you an email to <a>' . $email . '</a>. Go to your electronic mail to create your password.', 'adback-solution-to-adblock'); ?></i>
```

Comment:
This translation seems strange to me

Reply:
I also think you forgot to modify the translation file

Reply:
yep done